README.md

```diff
@@ -35,7 +35,7 @@ client. Thanks Andreas!
 
 confluent-kafka-dotnet is distributed via NuGet. We provide five packages:
 
-- [Confluent.Kafka](https://www.nuget.org/packages/Confluent.Kafka/) *[net45, netstandard1.3, netstandard2.0]* - The core client library.
+- [Confluent.Kafka](https://www.nuget.org/packages/Confluent.Kafka/) *[net462, netstandard1.3, netstandard2.0]* - The core client library.
```

Comment:
Not sure if this is a recommendation or a requirement, but if the latter the bump should probably go in the "Upgrade considerations" of the CHANGELOG, aight?

Reply:
yeah, i already added it there when making the change. i forgot that it was also listed in the README.